scripts/presubmit.sh

```diff
@@ -71,21 +67,19 @@ main() {
     shift 1
   done
 
-  local go_dirs="$(go list ./... | \
-    grep -v /vendor/)"
-  local go_srcs="$(find . -name '*.go' | \
-    grep -v mock_ | \
-    grep -v .pb.go | \
-    grep -v .pb.gw.go | \
-    grep -v _string.go | \
-    grep -v vendor/ | \
-    tr '\n' ' ')"
-  local proto_srcs="$(find . -name '*.proto' | \
-    grep -v vendor/ | \
-    tr '\n' ' ')"
+  cd "$(dirname "$0")"  # at scripts/
+  cd ..  # at /
```

Comment:
"at /" is wrong/misleading -- maybe "at top-level"

Reply:
Done.